adwords_user_list.go

```diff
@@ -67,8 +67,8 @@ type MutateMembersOperations struct {
 // Member holds user list member identifiers.
 // https://developers.google.com/adwords/api/docs/reference/v201708/AdwordsUserListService.Member
 type Member struct {
-	Email       string       `xml:"hashedEmail"`
-	MobileID    string       `xml:"mobileId"`
+	Emails      []string     `xml:"hashedEmail,omitempty"`
+	MobileIDs   []string     `xml:"mobileId,omitempty"`
```

Comment:
Something seems wrong with this change, according to the docs [MutateMembersOperand](https://developers.google.com/adwords/api/docs/reference/v201708/AdwordsUserListService.MutateMembersOperand) has an array of [Member](https://developers.google.com/adwords/api/docs/reference/v201708/AdwordsUserListService.Member) and [Member](https://developers.google.com/adwords/api/docs/reference/v201708/AdwordsUserListService.Member) dosn't have an array emails or mobile ID's but a single instance.

Reply:
It took me a while to notice it, but I think this line implies that they can be repeated `Multiple member identifiers of the same type for the same member can be provided at the same time.`

Based on that comment and the fact that the API accepts the data, I think the following should be allowed.
```
<membersList>
  <hashedEmail>...</hashedEmail>
  <hashedEmail>...</hashedEmail>
</membersList>
```

Reply:
Ah that makes sense, I was thinking in JSON terms where it would create an array of those members identifiers instead of the way the XML wil generate multiple member identifiers.

And if you’ve tested and it works LGTM!